index.html

```diff
@@ -48,13 +63,149 @@ <h3>LET'S CONNECT</h3>
+              <h1>Multi-post stories</h1>
+              <div class="canopy-div">
+                <p>Facebook</p>
+                <img src="Counter.svg" alt="" />
+                <p>Fullstack</p>
+                <img src="Counter.svg" alt="" />
+                <p>2015</p>
+              </div>
+             <p class="writeup-div">Experimental content creation feature that allows users to add to an 
+              existing story ovwr the cause of a day without spamming their friends
+             </p>
+             <button class="desktop-btn">HTML</button>
+             <button class="desktop-btn">CSS</button>
+             <button class="desktop-btn">JavaScript</button>
+
+             <p><button class="see-project">See Project</button></p>
+            </div>
+            <div class="mobile-image">
+              <img src="desktop3.svg" alt="" class="card-img-img">
+            </div>
+           </div>
+
+           <div class="mobile-version">
+            <div class="mobile-image">
+              <img src="desktopimg.svg" alt="" class="card-img-img">
+            </div>
+            <div class="mobile-section-div">
+              <h1>Facebook 360</h1>
+              <div class="canopy-div">
+                <p>facebook</p>
+                <img src="Counter.svg" alt="" />
+                <p>fullstack</p>
+                <img src="Counter.svg" alt="" />
+                <p>2015</p>
+              </div>
+             <p class="writeup-div">Experimental content creation feature that allows users to add to an 
+              existing story ovwr the cause of a day without spamming their friends</p>
+             <button class="desktop-btn">HTML</button>
+             <button class="desktop-btn">CSS</button>
+             <button class="desktop-btn">JavaScript</button>
+
+             <p><button class="see-project">See Project</button></p>
+            </div>
+           </div>
+
+           <div class="mobile-version">
+            <div class="mobile-section-div-left">
+              <h1>Uber Navigation</h1>
+              <div class="canopy-div">
+                <p>Uber</p>
+                <img src="Counter.svg" alt="" />
+                <p>Lead developer</p>
+                <img src="Counter.svg" alt="" />
+                <p>2018</p>
+              </div>
+             <p class="writeup-div">A daily selection of privately personalized reads; no accounts or sign ups required</p>
+             <button class="desktop-btn">HTML</button>
+             <button class="desktop-btn">CSS</button>
+             <button class="desktop-btn">JavaScript</button>
+
+             <p><button class="see-project">See Project</button></p>
+            </div>
+            <div class="mobile-image-right">
+              <img src="desktop2.svg" alt="" class="card-img-img">
+            </div>
+           </div>
+        </section>
+
+        <section class="desktop-second-about">
+          <div class="desktop-about-left">
```

Comment:
- [ ] I kindly suggest that you follow the Figma design by:
- [ ] Please kindly ensure that the horizontal spacing for the language, frameworks, skills are the same as the Figma design
- [ ] Please kindly make sure that your `Frameworks` is boldened just like the Figma design.
- [ ] Please kindly make sure that the JavaScript, HTML, and CSS are the same size as the Figma design
- [ ] Please kindly add the arrows on the right of the language, frameworks, and skills.


| Your Design  | Figma Design |
| ------------- | ------------- |
| <img width="310" alt="image" src="https://github.com/Developer-Kingz/my-portforlio-project/assets/52934128/9b041190-8cc6-4f18-ac4d-1dd2e0e3d723">  | <img width="425" alt="image" src="https://github.com/Developer-Kingz/my-portforlio-project/assets/52934128/48367daf-7f19-4a73-9d5c-e84ddbffaa61">  |